spec/cc/analyzer/issue_validations/content_validation_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "spec_helper"
+
+module CC::Analyzer::IssueValidations
+  describe ContentValidation do
+    describe "#valid?" do
+      it "is valid without any content" do
+        expect(ContentValidation.new({})).to be_valid
+      end
+
+      it "is valid with content with body" do
+        expect(ContentValidation.new("content" => { "body" => "hi"})).to be_valid
+      end
+
+      it "is not valid with content as string" do
+        expect(ContentValidation.new("content" => "hi")).not_to be_valid
+      end
+
+      it "is not valid with content as nil" do
+        expect(ContentValidation.new("content" => nil)).not_to be_valid
```

Comment:
So `db.smells.count({ content: { $exists: true, $in: [null] } })` is not going to run in a reasonable amount of time (my connection got broken by fab after an hour or so). Can't do `mapReduce` on slave (apparently), so I don't know of a good way to "background" the job. Any other ideas about how to query for this?